Formula/mariadb.rb

```diff
@@ -151,6 +151,9 @@ def caveats; <<-EOS.undent
 
     To connect:
         mysql -uroot
+
+    Using brew services will place a plist file at #{ENV["HOME"]}/Library/LaunchAgents/#{plist_name}
+    This plist file overrides the bind address to 127.0.0.1. Edit this plist file if you need to change the server's bind address.
```

Comment:
Do you mean putting it in `my.cnf` file in one of the locations? I think that would make it more obvious because most Google results I found about trying to change the MySQL bind address said to edit `my.cnf` in one of the proper locations. That doesn't work because the plist file overrides it. I figured it would be much simpler to just add some info to the caveat about how to fix the bind address if you need to.